src/containers/header/AppHeader.jsx

```diff
@@ -122,7 +122,7 @@ const Header = () => {
       }).unwrap()
 
       // update redux state with new data
-      dispatch(onProfileUpdate({ developerMode: !developerMode }))
+      dispatch(updateUserAttribs({ developerMode: !developerMode }))
```

Comment:
Because we are trying to use event based naming, maybe this should be `onDevModeChange` or `toggleDevMode` with a new action that only toggles the developerMode.

Reply:
Added toggleDevMode in user slice, used here.